service-deployment/prepare_hadoop_config.sh

```diff
@@ -56,7 +56,9 @@ cp hadoop/yarn-site.xml bootstrap/hadoop-service/hadoop-configuration/resourcema
 
 
 # patch for hadoop-run
-patch src/hadoop-run/capacity-scheduler.xml config-patch/capacity-scheduler.xml.patch
+# It is not necessary to patch capacity-scheduler.xml here because this file
+# will be prepared by deploy.py during the process of service deployment.
+#patch src/hadoop-run/capacity-scheduler.xml config-patch/capacity-scheduler.xml.patch
```

Comment:
@fanyangCS Should we directly put scheduler.xml to the hadoop-configuration. Or make it as a patch?

Reply:
yes. make it a patch.

---
In reply to: [177291209](https://github.com/Microsoft/pai/pull/412#discussion_r177291209) [](ancestors = 177291209)